android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java

```diff
@@ -129,14 +129,14 @@ private Notification buildNotification() {
         final int notices;
         final String projectName;
 
-        nb = (android.support.v7.app.NotificationCompat.Builder)(new NotificationCompat.Builder(this.context).
-            setContentTitle(this.context.getResources().getQuantityString(
-                R.plurals.notice_notification,
-                notices = this.currentlyNotifiedNotices.size(),
-                projectName = this.currentlyNotifiedNotices.get(0).project_name,
-                notices)).
-            setSmallIcon(R.drawable.mailw).
-            setContentIntent(this.contentIntent));
+		nb = new NotificationCompat.Builder(this.context, "main-channel");
```

Comment:
@adamradocz, please fix indents

Reply:
Resolved. I can't believe it always happens. :)

Reply:
Default setting in your editor? I use Notepad++, and after being prompted by similar requests, I found it in Settings --> Preferences --> Language --> Tab settings: Replace by space

Reply:
What if we use the tab as standard?

Reply:
I think you'd get a howl of protest. There is actually a stated common standard, which helps people navigate around in a comfortable environment:

https://boinc.berkeley.edu/trac/wiki/CodingStyle#indent

Reply:
Thanks